search-rels.md

```diff
@@ -1,14 +1,19 @@
-# Endpoint' Search methods
+# Search methods on collection endpoints
 [REST Overview Documentation](README.md)
 
-Resources collection should expose search methods to return specific subsets depending on the user requirements. Such as the items from a specific submitter, the top level communities, etc.
-All these methods should be exposed under <resources-collection-endpoint>/search/<method-name> at <resources-collection-endpoint>/search a HAL document should list all the available search methods for the resources collection
+Individual collection endpoints (e.g. `/api/core/{model}`) may expose search methods (`/api/core/{model}/search`) to return specific subsets of resources. This allows for filtering of the collection (group of resources) to only return specific resources that match the search requirements. Some examples include a search endpoint for only returning top-level Communities (e.g. `/api/core/communities/search/top`) or a search endpoint for only returning WorkspaceItems from a specific Submitter (e.g. `/api/submission/workspaceitems/search/findBySubmitter?uuid=<:submitter-uuid>`).
 
-**Please note that the [Discovery search](search-endpoint.md) is a completely separate topic**
+Search endpoints on a collection of resources should act as follows:
+* All available search methods (for the given resource) should be exposed under `/api/core/{model}/search`. The result should be a HAL document.
+* Individual search methods should be exposed under `/api/core/{model}/search/{method-name}`
 
-## Endpoint that have Search methods
-* /core/communities
-* /config/submissiondefinitions
+**Please note that the [Discovery search](search-endpoint.md) is a completely separate topic**
 
-## Under Development
-* /submission/workspaceitems
\ No newline at end of file
+## Endpoints that have Search methods
+* [/api/core/communities](communities.md)
+* [/api/core/metadatafields/](metadatafields.md)
+* [/api/config/submissiondefinitions](submissiondefinitions.md)
+* [/api/submission/workspaceitems](workspaceitems.md)
+* [/api/workflow/claimedtasks](claimedtasks.md)
+* [/api/workflow/polltasks](polltasks.md)
+* [/api/workflow/workflowitems](workflowitems.md)
```

Comment:
Perhaps not in this Pull Request, but we will need to add a reference to this contract: https://github.com/DSpace/Rest7Contract/blob/configurable_entities/relationships.md

Reply:
@paulo-graca : yes, we'd include that in this list *after* the Configurable Entities work is added to the `master` branch.  Currently, that `relationship` endpoint only exists on the `configurable_entities` branch, and not on `master` (and this PR we're talking about here is for `master`).